extensions/amp-story/1.0/story-analytics.js

```diff
@@ -78,6 +79,12 @@ export class StoryAnalyticsService {
     /** @const @private {!./variable-service.AmpStoryVariableService} */
     this.variableService_ = getVariableService(win);
 
+    /** @private {!Object} */
+    this.eventsPerPage_ = map();
```

Comment:
Could you please add a typedef? It's pretty annoying to write but it'll pay off in the long term